PR-App/PR-App/Screens/UserMedicationInfoVC.swift

```diff
@@ -64,14 +69,20 @@ extension UserMedicationInfoVC: UICollectionViewDataSource, UICollectionViewDele
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as! CustomCell
-        cell.addButtonImage.setImage(UIImage(systemName: "plus.circle.fill"), for: .normal)
-        cell.addNewMedsTitle.text = "Add medication"
+        cell.imageCell.image = UIImage(systemName: "plus.circle.fill")
+        cell.newMedsTitle.text = "Add medication"
         return cell
     }
     
     func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-        let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: CustomHeader.reuseID, for: indexPath)
+        let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: CustomCellHeader.reuseID, for: indexPath)
         
         return header
     }
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let destVC = UserMedicationDetailVC()
```

Comment:
change destVC to meaningful name, for example:
`destinationViewController` or `UserMedicationDetailVC`

Reply:
Done